dev_guide/builds.adoc

```diff
@@ -134,6 +135,55 @@ of a missing `save-artifacts` script.
 For information on how to create a builder image supporting incremental builds
 see the link:../creating_images/s2i.html[creating images guide].
 
+=== Override builder image scripts
+You can override the scripts (`assemble`, `run`, `save-artifacts`) provided by the builder image in one of two ways: Either provide an `assemble`, `run`, and/or `save-artifacts` script in the `.sti/bin` directory of your application source repository, or provide a url of a directory containing the scripts as part of the strategy definition:
```

Comment:
Maybe it would be good to add that value from `scripts` field will take precedence over `.sti/bin` in case where there's one of the scripts available in both places.